### ISSUE TITLE
Poor screen reader compatibility

### ISSUE THREAD

**Issue by LiamOSullivan** [BuildingCityDashboards/bcd-dd-v2.1#23]
The whole site should be accessible to screen readers. This should be assessed using  an industry standard benchmarking tool
